pkg/k8s/k8s.go

```diff
@@ -473,17 +473,17 @@ func CmdDelK8s(ctx context.Context, c calicoclient.Interface, epIDs utils.WEPIde
 		}
 	}
 
-	// Release the IP address for this container by calling the configured IPAM plugin.
-	logger.Info("Releasing IP address(es)")
-	ipamErr := utils.DeleteIPAM(conf, args, logger)
-
 	// Clean up namespace by removing the interfaces.
 	logger.Info("Cleaning up netns")
 	err = utils.CleanUpNamespace(args, logger)
 	if err != nil {
 		return err
 	}
 
+	// Release the IP address for this container by calling the configured IPAM plugin.
+	logger.Info("Releasing IP address(es)")
+	ipamErr := utils.DeleteIPAM(conf, args, logger)
+
 	// Return the IPAM error if there was one. The IPAM error will be lost if there was also an error in cleaning up
```

Comment:
This comment isn't correct any more - I think we can just remove it?

Also, we can rename `ipamErr` to `err` now since we're not dealing with two errors at a time.